crux-http-server/src/crux/http_server.clj

```diff
@@ -370,7 +370,7 @@
   (close [_]
     (.stop server)))
 
-(defn start-http-server
+(defn ^:deprecated start-http-server
```

Comment:
I was thinking about this - could we just remove the fn? What's the benefit in keeping - forcing users to update code during alpha isn't so bad?

Reply:
one for a quick discussion, I reckon - I'd be tempted to keep this in for one overlap release so that people can migrate easier? not wedded to that, though.

Reply:
gonna merge this for now, and add this discussion to #580